.env.test

```diff
@@ -0,0 +1,8 @@
+MONGODB_URI=mongodb://localhost:27017/test
```

Comment:
I was looking at the `dotenv` [documentation](https://github.com/motdotla/dotenv#should-i-commit-my-env-file) and noted that they recommend against committing `.env` files, as well as having multiple `.env` files.

Correct me if I'm wrong, but it seems as though you're using `dotenv` to change the application depending on what environment you're in. If so, you might consider using [`cross-env`](https://www.npmjs.com/package/cross-env) to set `process.env.NODE_ENV` and using conditional statements around that.